docs/using/dashboard.md

```diff
@@ -0,0 +1,42 @@
+---
+title: The Infra Dashboard
+position: 3
+---
+
+# The Infra Dashboard
+
+The **Dashboard** is the main UI in Infra, providing easy access to everything an administrator needs to do with **Infra**. It is split into several main sections, accessible with the menu on the left, or if the window is narrow enough, when you click the three line icon, also known as a hamburger icon.
+
+## Infrastructure
+
+Infrastructure is where you go to configure your Kubernetes Cluster, and to configure grants of users and roles to those clusters.
+
+- [Go to the Kubernetes Connector page to learn about clusters](../manage/connectors/kubernetes.md).
+- [Go to the Granting Access page to learn about grants](../manage/grants.md).
+
+## Providers
+
+Providers is where you go to configure your Identity Providers, such as Okta, Azure, and Google.
+
+- [Go to the Okta Identity Provider page to learn how to configure Okta with Infra](../manage/idp/okta.md)
```

Comment:
Can we make this a bullet list?

```
* [Okta](../manage/idp/okta.md)
* ...
```

Reply:
`-` and `*` are equivalent. Both create unordered lists